src/components/Composer.vue

```diff
@@ -632,7 +632,7 @@ export default {
 
 			this.state = STATES.UPLOADING
 
-			return this.attachmentsPromise
+			this.attachmentsPromise
```

Comment:
I think you want to `await` sending the message. right now you trigger they sync just 500ms after *starting the send request*